plugins/network-elements/opendaylight/src/main/java/org/apache/cloudstack/network/opendaylight/OpendaylightElement.java

```diff
@@ -156,7 +153,7 @@ public HostVO createHostVOForDirectConnectAgent(HostVO host, StartupCommand[] st
 
     @Override
     public DeleteHostAnswer deleteHost(HostVO host, boolean isForced, boolean isForceDeleteStorage) throws UnableDeleteHostException {
-        return new DeleteHostAnswer(true);
+        return null;
```

Comment:
@syed I am not sure if this is used or maintained by anyone. But the right fix may be to use some checks to return a valid answer or null. How is this handled in some of the other network elements?